controllers/balance-widget.controller.es6

```diff
@@ -20,6 +21,10 @@ export default class BalanceWidgetController {
     this.showRefreshButton = false;
     this.accountNotFound = false;
 
+    this.checkAddressAvailable = session.data && session.data['useLedger'] && !session.data['ledgerAppVersion'].startsWith('1');
```

Comment:
We should use more sophisticated check then `startsWith`. It's possible that you will release `10.x.x` some day and this will stop working.